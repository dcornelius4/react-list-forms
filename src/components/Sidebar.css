.Sidebar {
  display: flex;
  justify-content: space-around;
  flex-direction: column;
  height: 20em;
  width: 50%;

  background-color: darkred;
  color: burlywood;
}