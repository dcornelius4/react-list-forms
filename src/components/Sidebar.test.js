import React from 'react';
import { shallow } from 'enzyme';
import Sidebar from './Sidebar';

describe('Sidebar Component', () => {
  it('renders a sidebar', () => {
    const wrapper = shallow(
      <Sidebar>
        <a href="#">Test Snapshot</a>
      </Sidebar>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
