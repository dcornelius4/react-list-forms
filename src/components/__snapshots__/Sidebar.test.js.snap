// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar Component renders a sidebar 1`] = `
<nav
  className="Sidebar"
>
  <a
    href="#"
  >
    Test Snapshot
  </a>
</nav>
`;
